sig-cli/CONTRIBUTING.md

```diff
@@ -396,13 +381,14 @@ See the sig-cli [community page] for points of contact and meeting times:
 - `@kubernetes/sig-cli-proposals` flags a design proposal.
```

Comment:
As I said before, I don't think any of these `@kubernetes/sig-cli-*` exist anymore, or at least are not used.  Maybe someone else can tell me I'm wrong, but they don't seem to resolve when I type them in a github comment.

What I **do** see are the following, but even then, I'm not sure what these would do:
* `@kubernetes/sig-cli-leads`
* `@kubernetes/sig-cli-kubectl-maintainers`